openlcsd/flow/tasks.py

```diff
@@ -366,6 +368,44 @@ def download_shared_remote_source(context, engine):
         )
 
 
+def download_maven_source_from_corgi(context, engine, tmp_dir):
+    # deal with MAVEN component
+    component = context.get('component')
+    download_url = component.get('download_url')
+    if not download_url:
+        raise RuntimeError(f"component {component['purl']} download_url empty")
+    name_list = list_http_files(download_url)
+    pom_filename, jar_filename, sources_jar_filename = None, None, None
+    for name in name_list:
+        if name.endswith(".pom"):
+            pom_filename = name
+        if name.endswith(".jar") and "doc" not in name:
```

Comment:
`The 'doc' not in name` excludes all jar files with 'doc' in its name including xdocx.jar, is this expected? could you add a comment for this line please?

Reply:
That's not excepted. Thanks for your review, i will change the code